### PR TITLE
Add ember-bootstrap writeup

### PR DESCRIPTION
**Pull request by amyrlam**

### CODE REVIEW

File: source/2020-08-07-the-ember-times-issue-159.md [hunk 1]
```diff
@@ -25,14 +29,14 @@ READMORE
...
 ---
 
-## [Section title in sentence case 🐹](section-url)
+<!--alex ignore just-->
+## [Just released: stable version of ember-bootstrap v4 💥](https://twitter.com/simonihmig/status/1290710885070262275)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Simon Ihmig (@simonihmig)](https://github.com/simonihmig) announced the release of the **stable** version of ember-bootstrap v4, [tweeting](https://twitter.com/simonihmig/status/1290710885070262275) that it's another step toward a full Octane refactoring! Glimmer components are coming next.
```
Comment (ijlee2) [inline source/2020-08-07-the-ember-times-issue-159.md]: Maybe we can shorten the phrase?
```suggestion
[Simon Ihmig (@simonihmig)](https://github.com/simonihmig) announced the **stable release** of ember-bootstrap v4, [tweeting](https://twitter.com/simonihmig/status/1290710885070262275) that it's another step toward a full Octane refactoring! Glimmer components are coming next. 
```

File: source/2020-08-07-the-ember-times-issue-159.md [hunk 2]
```diff
@@ -25,14 +29,14 @@ READMORE
...
+[Simon Ihmig (@simonihmig)](https://github.com/simonihmig) announced the release of the **stable** version of ember-bootstrap v4, [tweeting](https://twitter.com/simonihmig/status/1290710885070262275) that it's another step toward a full Octane refactoring! Glimmer components are coming next. 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Bootstrap users can check out the full list of changes in the [changelog](https://github.com/kaliber5/ember-bootstrap/blob/master/CHANGELOG.md). For breaking changes, give [ember-bootstrap-codemods](https://github.com/kaliber5/ember-bootstrap-codemods) a try. We are appreciative of all the codemods in the Ember community!
+
+Not familiar with ember-bootstrap? It's an ember-cli addon that provides a collection of native Ember components that mimic the [original Bootstrap](https://getbootstrap.com/) plugins and components in an Ember-friendly way, replacing the need for `bootstrap.js`. For more info, check out the [website](https://www.ember-bootstrap.com/) and the corresponding [repo](https://github.com/kaliber5/ember-bootstrap).
```
Comment (ijlee2) [inline source/2020-08-07-the-ember-times-issue-159.md]: The 2nd sentence felt a bit long, but I wasn't sure how I might break it into two sentences.

I wasn't sure what you meant by native Ember components (are they any different from Ember components?) so I removed the word in the suggestion below.
```suggestion
Not familiar with ember-bootstrap? It's an Ember CLI addon that provides a collection of Ember components that mimic the [original Bootstrap](https://getbootstrap.com/) plugins and components in an Ember-friendly way, replacing the need for `bootstrap.js`. For more info, check out the [website](https://www.ember-bootstrap.com/) and the corresponding [repo](https://github.com/kaliber5/ember-bootstrap).
```

**Review by ijlee2 — APPROVED** [review]
Your writeup of the v4 announcement read great! Thanks for covering the news!

**Comment by amyrlam** [comment]
@ijlee2 thanks for the review, super helpful! Agree sounds better 💯

**Pull request merged**